### PR TITLE
image update

### PR DESCRIPTION
**Pull request by mbruns91**
updated basemimage in ROOT_CONTAINER in docker-stacks/base and environment.yml for all images.

### CODE REVIEW

File: base/environment.yml [hunk 1]
```diff
@@ -2,5 +2,7 @@ channels:
 - conda-forge
 dependencies:
 - ipywidgets
-- pyiron_base =0.5.7
-- seaborn =0.11.2
+- pyiron_base =0.5.31
```
Comment (jan-janssen) [inline base/environment.yml:5]: Can we already use `pyiron_base=0.5.32` that should fix the SQLalchemy issues? Or we temporarily enforce `sqlalchemy<2.0.0`. https://github.com/pyiron/pyiron_atomistics/pull/949

Comment (mbruns91) [reply base/environment.yml:5]: Yeah, sure. I edited this file before the sqlalchemy issue arised and forgot to check again... My bad, sorry.

Comment (jan-janssen) [reply base/environment.yml]: If you also need `pyiron_atomistics` then please add `sqlalchemy<2.0.0`. `pyiron_base` is currently backwards compatible https://github.com/conda-forge/pyiron_base-feedstock/pull/147

Comment (mbruns91) [reply base/environment.yml]: I'll do that 👍

File: base/environment.yml [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
 dependencies:
+- python =3.9
 - ipywidgets
-- pyiron_base =0.5.7
-- seaborn =0.11.2
+- pyiron_base =0.5.32
+- sqlalchemy <2.0.0
+- seaborn =0.12.2
+- damask
+- openssh
```
Comment (jan-janssen) [inline base/environment.yml:10]: The `pyiron_base` environment should be minimal. So it should not contain specific tools like `damask`

**Comment by mbruns91** [comment]
We just tested it on a similar server as will be used on Wednesday, everything seemed to be working fine 👍

**Pull request closed without merging**